common_features.mk

```diff
@@ -144,6 +144,9 @@ endif
     SRC += $(QUANTUM_DIR)/rgb_matrix.c
     SRC += $(QUANTUM_DIR)/rgb_matrix_drivers.c
     CIE1931_CURVE = yes
+    ifeq ($(strip $(RGB_MATRIX_KEYPRESSES)), yes)
```

Comment:
Just curious, why is this added/moved to here? 
Usually, we have the config.h file for settings like this.

Reply:
I can remove it, added it as a hack tbh

Reply:
Yeah, I think it would be for the best to remove it